### PR TITLE
fix: add git to aarch64 container in CI

### PR DESCRIPTION
**Pull request by lemire**
We need git for the tests to run.

### CODE REVIEW

**Review by anonrig — APPROVED** [review]

**Comment by lemire** [comment]
Hmmm... It is failing due to a timeout.

So maybe we should leave it as it is. It is currently not running the tests, but, at least, it is testing that we can build the software.

**Comment by CarlosEduR** [comment]
Nice we merged the workflow, now I intend to take a better look about why it's not running the tests correctly.

**Comment by anonrig** [comment]
I think we actually have a problem that causes Gtest child process to get killed.

**Comment by lemire** [comment]
:-/

**Comment by anonrig** [comment]
I think we can land this as it is, since it is fixing the timeout issue.

**Comment by lemire** [comment]
@anonrig Do you why it is failing?

**Comment by lemire** [comment]
Yeah. We have a problem building with the sanitizers. It does not seem to work, or it produces a binary that cannot run.

**Comment by lemire** [comment]
There is definitively something wrong. I tried to simplify the build, but it is still broken.

I don't think it is us.

**Comment by anonrig** [comment]
I think the issue is the sanitizer

**Comment by lemire** [comment]
> I think the issue is the sanitizer

Yes. Once I removed it, then the tests passed.

**Comment by lemire** [comment]
Let me try something else.

**Comment by lemire** [comment]
I have reenabled sanitizers but with a different distro. We are currently using Ubuntu Noble which is maybe too recent.

**Comment by lemire** [comment]
hmmmm... we get a *different* error.

File: tests/CMakeLists.txt [hunk 1]
```diff
@@ -39,12 +39,13 @@ else()
...
   target_link_libraries(url_search_params PRIVATE simdjson GTest::gtest_main)
 
-  gtest_discover_tests(wpt_tests)
-  gtest_discover_tests(url_components)
-  gtest_discover_tests(basic_tests)
-  gtest_discover_tests(from_file_tests)
-  gtest_discover_tests(ada_c)
-  gtest_discover_tests(url_search_params)
+  gtest_discover_tests(wpt_tests PROPERTIES TEST_DISCOVERY_TIMEOUT 600)
+  gtest_discover_tests(wpt_tests DISCOVERY_MODE PRE_TEST)
```
Comment (anonrig) [inline tests/CMakeLists.txt:43]:
```suggestion
```

**Comment by lemire** [comment]
@anonrig Have a look!!!!!!!!!!!!!!!!!!

**Comment by CarlosEduR** [comment]
nice work @lemire!

**Comment by lemire** [comment]
The trick is to run with bound checks. It should catch most memory problems.

**Review by anonrig — APPROVED** [review]
amazing news!

**Pull request merged**